### DISCUSSION TITLE
Project roadmap / automated devops?

### DISCUSSION THREAD

**Discussion by ayushin** [modoboa/modoboa#2328] [General]
Hello team!

Really happy to see this project exists and it would be really awesome if one day it could get to the maturity and user-base size of mailcow-dockerized which we are using at the moment.

We love Django and I think potentially this gives a chance to have much more flexibility and faster development short and long term.

One issue I see deficient is lack of a modern docker / helm automatic deployment (perhaps even combined with terraform) 

Are there any plans to work on that?

**Comment by tonioo** [comment]
@ayushin No particular plan for this right now but you're free to contribute if that's something you know and and you're willing to do :)

**Comment by UliPrantz** [comment]
@ayushin @tonioo same here! As I evaluated the options of self-hostable email servers in the last days the lack of an all-in-one solution (so docker mailserver is already out) that doesn't need root rights or rely on wild update scripts (which mailcow-dockerized does) is almost nonexistent.

I thought modoboa actually would be a perfect solution as I saw the docker-compose.yaml file but turns out it's only for development purposes. I could try using the installer script and the development setup in docker-compose.yaml to get a first production-ready docker-compose.yaml up and running, if someone points me in the right direction!

Also as the thread is from 2021 is this something that is on the roadmap or even desirable or are there any other plans - easy setup and installation seem to be a great driver for widespread adoption.
